### PR TITLE
feat(web-app): add preview mdx component

### LINKED ISSUE #613: Preview

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #613

#### Testing / reviewing

http://localhost:3000/mdx/preview
http://localhost:6006/?path=/docs/mdx-preview--default
https://carbondesignsystem.com/developing/react-tutorial/step-4/

### CODE REVIEW

**Review by francinelucca — DISMISSED** [review]
🚀 
image

**Review by francinelucca — DISMISSED** [review]

File: services/web-app/components/preview/preview.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import styles from './preview.module.scss'
+
+/**
+ * The `<Preview>` component is a simple wrapper for an `<iframe />` with
+ * styling added to allow it to display responsively within the Platform.
+ */
+const Preview = ({ title, ...props }) => (
+  <iframe loading="lazy" title={title} {...props} className={styles.preview} />
+)
+
```
Comment (francinelucca) [inline services/web-app/components/preview/preview.js:17]:
```suggestion

Preview.propTypes = {
  title: Proptypes.string
}

```

File: services/web-app/stories/mdx/preview.stories.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react'
+
+import Preview from '@/components/preview'
+
+const stories = {
+  title: 'MDX/Preview',
+  component: Preview,
+  parameters: {}
```
Comment (francinelucca) [inline services/web-app/stories/mdx/preview.stories.js:14]:
```suggestion
  parameters: {},
  argTypes: {
    title: {
      control: false
    }
  }
```

**Review by francinelucca — CHANGES_REQUESTED** [review]
Looks good, minor suggestions

**Review by francinelucca — DISMISSED** [review]

**Review by andreancardona — DISMISSED** [review]

**Review by francinelucca — DISMISSED** [review]
🎉

**Review by andreancardona — APPROVED** [review]
🔥

**Review by francinelucca — APPROVED** [review]

**Pull request merged**